config.default

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Another thing comes into my mind: how we'll store and update the configuration file on Heroku?

Should not we just keep the real (production) config in the repository (there is nothing secret in it)? In future, we maybe even will setup automatic deploy by pushing to master.

I like this flow: this way we can track updates. So we'll know if something is forgotten or was updated in a wrong way.

Reply:
Yes, the configuration file is stored in the repository when the service is hosted on Heroku.

Reply:
So it supposed to be our real production configuration, right? Should not we add 'release' repository kind so?

Reply:
Add 'release' to configuration.